source/library/CabalGild/Unstable/Extra/Name.hs

```diff
@@ -21,8 +22,10 @@ isIf :: Fields.Name a -> Bool
 isIf = (== String.toUtf8 "if") . value
 
 -- | Returns true when the name is @"elif"@, false otherwise.
-isElif :: Fields.Name a -> Bool
-isElif = (== String.toUtf8 "elif") . value
+isElif :: CabalSpecVersion.CabalSpecVersion -> Fields.Name a -> Bool
+isElif csv n =
+  csv >= CabalSpecVersion.CabalSpecV2_2
+    && value n == String.toUtf8 "elif"
```

Comment:
The `elif` section only became a thing with Cabal 2.2. So for versions before that, we no longer try to format it. Hopefully this will help people realize that something is wonky.